charts/seed-bootstrap/templates/aggregate-prometheus/allow-from-aggregate-prometheus-network-policy.yaml

```diff
@@ -18,8 +18,16 @@ spec:
           app: aggregate-prometheus
           role: monitoring
     ports:
-    - port: metrics
+    - port: {{ .Values.global.elasticsearchPorts.metricsExporter }}
       protocol: TCP
+    - port: {{ .Values.fluentd.ports.metrics }}
+      protocol: TCP
+    - port: 9569 #hvpa
```

Comment:
This prometheus does not scrape hvpa or the vpa so why do we need to add this to the network policy? Prometheus may scrape them in the future but it can be added then.

Reply:
@wyb1 port `metrics` is already defined in these places in the chart.

Reply:
+1 to what @wyb1 mentioned.

@rootfs you are right but I think this kind of openness was not intended. At the moment I can only find the `fluent-es` pods carrying the label `networking.gardener.cloud/from-aggregate-prometheus: allowed`. So maybe it's even enough to use `{{ .Values.global.fluentd.ports.metrics }}` at the moment?